jobs/db_store.go

```diff
@@ -338,11 +348,32 @@ func (s *DatabaseStore) scanParams(method string, params []byte) (interface{}, e
 	}
 }
 
+func (s *DatabaseStore) scanResult(method string, result []byte) (interface{}, error) {
+	if result == nil {
+		return nil, nil
```

Comment:
Unmarshalling `[]byte(nil)` results in an empty value, so this is actually _less_ code than before :1st_place_medal: